pkgs/applications/science/logic/surelog/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python3
+, pkg-config
+, libuuid
+, openjdk11
+}:
+
+stdenv.mkDerivation rec {
+  pname = "Surelog";
+  version = "2022.05.15";
+
+  src = fetchFromGitHub {
+    owner = "chipsalliance";
+    repo = pname;
+    rev = "15d3698ca5c7d45dd95b58c15e76131420cb001c";
+    hash = "sha256-dfje9yZ8ZR7x1EUxDUpKNcOWKYTPwPG6T4HzudV59R4=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libuuid
+    openjdk11
+    (python3.withPackages (p: with p; [
+      psutil
+      orderedmultidict
+    ]))
+  ];
+
+  postInstall = ''
+    mv $out/lib/surelog/* $out/lib/
```

Comment:
there is also a pkg/ directory, which we should probably leave in lib/surelog (it is some pre-generated parsed UVM code).

The libflatbuffers and antlr runtime library it also creates is a little annoying, I'll try to upstream a change in which these are taken from the system instead of if available.